README.md

```diff
@@ -30,13 +94,9 @@ nvidia-docker run \
     --volume /absolute/path/to/data:/app/data \
```

Comment:
we should mention (or replace? windows users can not use this) `--volume "$( readlink data -f )":/app/data`